### ISSUE TITLE
Migrate Place to either Event Place or Publisher Place

### ISSUE THREAD

**Issue by rmzelle** [citation-style-language/zotero-bits#6]
"Place"/**place** is currently always mapped to both **event-place** and **publisher-place** in CSL. This is problematic as a) CSL styles can’t distinguish between the two variables and b) some item types would benefit from having both **eventPlace** and **publisherPlace** fields.

"Place"/**place** should probably be migrated to "Event Place"/**eventPlace** for _audioRecording_, _presentation_, _radioBroadcast_, _tvBroadcast_ and _videoRecording_, and to "Publisher Place"/**publisherPlace** for _book_, _bookSection_, _computerProgram_, _conferencePaper_, _dictionaryEntry_, 
_encyclopediaArticle_, _hearing_, _manuscript_, _map_, _newspaperArticle_, _patent_, _report_ and _thesis_. The item types _conferencePaper_ and _newspaperArticle_ require both fields (http://forums.zotero.org/discussion/8684/).

"Event Place"/**eventPlace** should be added to _letter_, which currently has no place field.

**Comment by tillheilmann** [comment]
For what it’s worth, I think interview and song should also have an event-place field:

- interview (event-place for the place where the interview took place, duh.)
- song (event-place could/should be used to designate the place where the recording took place, e.g. "The Beatles. Sgt. Pepper's Lonely Hearts Club Band. London, EMI. 1969. Abbey Road Studios" where "Abbey Road Studios" is event-place and "London" is publisher-place).

As has been said by others before, speech and graphic/Artwork need an event-place as well. (graphic/Artwork definitely needs one for pieces of performance art, for example.)

**Comment by tillheilmann** [comment]
This is the list of suggested changes to the mapping of place I came up with when I tried to get my head around it. I made some proposals and still have several questions. Maybe it is of some help to you guys.

artwork
- "Event Place"/event-place: This could be used for the place where the artwork was produced. It is, of course, not the same as archive-place and is necessary for performance art pieces and the like.

audioRecording
- "Event Place"/event-place: This would be where the recording actually took place (e.g. Abbey Road Studios, Lincoln Center etc.)
- "Publisher Place"/publisher-place: This would be the place of the record label or company.

book
- "Publisher Place"/publisher-place

bookSection
- "Publisher Place"/publisher-place

computerProgram
- "Publisher Place"/publisher-place

conferencePaper
- "Publisher Place"/publisher-place
- "Event Place"/event-place: This could be used for the place where the conference took place, which is not necessarily the same as the place where the paper/proceedings were published.

dictionaryEntry
- "Publisher Place"/publisher-place

encyclopediaArticle film
- "Publisher Place"/publisher-place

hearing
- "Publisher Place"/publisher-place: ?? (see below)
- "Event Place"/event-place: I am not sure about the use of this in the US context. But it seems to me that a hearing is an event, and event-place should be used. Whether publisher-place is needed additionally to give the place where the publication of the document was produced I do not know.

interview
- "Event Place"/event-place: This could be where the interview took place. This seems very useful to me.

letter
- "Event Place"/event-place: Is this supposed to be the place where the letter was sent FROM or TO?

manuscript
- "Publisher Place"/publisher-place: I don’t know about this one. I thought manuscript was to be used for unpublished documents. (That is pretty much the dictionary definition of the word, at least.)

map
- "Publisher Place"/publisher-place

newspaperArticle
- "Publisher Place"/publisher-place
- "Event Place"/event-place: Is this supposed to be the place where the correspondent is reporting from?

patent
- "Publisher Place"/publisher-place

podcast
- "Event Place"/event-place: This could be used for the place where the podcast was recorded. I am not sure about it, but it could be useful to have it.

presentation
- "Event Place"/event-place

radioBroadcast
- "Publisher Place"/publisher-place: ?? (see below)
- "Event Place"/event-place: A broadcast is certainly an event, but I expect people to cite the place where the station or network has its headquarters. And for this, publisher-place seems the right choice. However, a show can be broadcast from a different place, and this would then be event-place.

report
- "Publisher Place"/publisher-place

thesis
- "Publisher Place"/publisher-place

tvBroadcast (same as radioBroadcast)
- "Publisher Place"/publisher-place: ?? (see below)
- "Event Place"/event-place: A broadcast is certainly an event, but I expect people to cite the place where the station or network has its headquarters. And for this, publisher-place seems the right choice. However, a show can be broadcast from a different place, and this would then be event-place.

videoRecording
- "Event Place"/event-place: A video recording recorded an event which took place somewhere (or in more places than one), alright. But my German localization of Zotero calls place "Ort" and publisher "Studio" which suggests that place, as it used now, means place of the publisher. Should this reallly be mapped to event-place?

**Comment by bwiernik** [comment]
For migrating current usage of "Place" in Zotero, only presentation should move "Place" to "event-place". (letter doesn't currently have a "Place" field). All other "Place" fields should generally be assumed to be "publisher-place". In some cases (e.g., conferencePaper, perhaps videoRecording and interview) a separate "Event Place" field should be added.

**Comment by dstillman** [comment]
Do we want to say "Publisher Place" and "Event Place" even on item types that only have one or the other? I'd prefer to just say "Place" at least for "Publisher Place" when that's the only option, since we've had that for the last 15 years, and it's right next to Publisher.

Somewhat related question: if you convert an item type with Publisher Place to an item type with only Event Place, would you expect the value to transfer, even though it's technically a different field?

**Comment by dstillman** [comment]
A belated thanks to @tillheilmann for the write-up. If anyone has any comments on or disagreements with that summary, please let me know.

Note that it questions the use of Publisher Place for Hearing, which currently has Place. Either Hearing values need to be migrated to Event Place or it needs both.

**Comment by adam3smith** [comment]
For hearing: I think it's logical to switch to event-place and we don't need two places: at least in the US, hearings are typically cited without any location (e.g. Bluebook, APA), but having one field available for other/more flexible cases makes sense.

**Comment by adam3smith** [comment]
Other comments:
One thing I'd want to look at a bit closer is if artwork also needs publisher place (we definitely want event I think) or if indeed everything is covered between archive and event.

Only other small disagreement:
I wouldn't add event-place to broadcasts. I've never seen that. Following that logic, I'd suggest using publisher place for podcasts: podcasts are increasingly similar to radio broadcasts,  with traditional publishers like NPR, gimlet, TAL, etc. and having a publisher makes more sense for consistency. I'd guess this will be rarely cited, but good to have.

**Comment by bdarcus** [comment]
Just for clarification, where does JSON, and the 1.1 work, fit here in general?

Mainly just reminding you all.

**Comment by adam3smith** [comment]
Can you say more? I might have missed a part of the 1.1. discussion, but this is just mapping Zotero fields to existing CSL, so I'm not sure how 1.1/JSON would matter here?

**Comment by bwiernik** [comment]
@dstillman I have some disagreements with @tillheilmann noted above.

My summary:
- Place should be mapped to `publisher-place` for all item types except Presentation (`speech`) and Hearing (`hearing`)
- Place should be replaced with Event Place (`event-place`) for Presentation (`speech`) and Hearing (`hearing`)
- Event Place (`event-place`) should be added to Conference Paper (`paper-conference`), in addition to Place (`publisher-place`)
- Both Event Place (`event-place`) and Place (`publisher-place`) should be added to Interview (`interview`)

Keeping "Place" (`publisher-place`) makes sense to me. I think we should always label `event-place` as "Event Place" for clarity. For Conference Paper and Interview, it might be good to label it as "Publisher Place", but if the fields are "Place" (next to Publisher) and "Event Place", I think that would be clear enough too.

**Comment by bdarcus** [comment]
No worries; I just wasn't clear.

**Comment by dstillman** [comment]
Does Interview need Publisher, if it's getting Publisher Place?

**Comment by bwiernik** [comment]
Yes.

**Comment by dstillman** [comment]
Following up on @tillheilmann's question above, can a manuscript have a Publisher?

**Comment by bwiernik** [comment]
Yes. The institution where a manuscript was produced is cited in APA style.

**Comment by dstillman** [comment]
Should we call it "Institution" or "Publisher"?

**Comment by bwiernik** [comment]
Institution

**Comment by dstillman** [comment]
https://forums.zotero.org/discussion/98078/place-of-publication-for-item-type-film

Did we want either place for `film`?

**Comment by adam3smith** [comment]
Film should have place --> publisher-place, yes. It's routinely cited, e.g. Chicago

Cuarón, Alfonso, dir. Gravity. 2013; Burbank, CA: Warner Bros. Pictures, 2014. Blu-ray Disc, 1080p HD.

(yes, that's technically a video recording, but arguably every film you watch is a recording of some sort)

**Comment by dstillman** [comment]
Great, thanks.
